src/lib.rs

```diff
@@ -168,32 +481,106 @@ impl Child {
         }
     }
 
-    pub fn kill(&self) -> Result<()> {
+    /// Returns process identifier associated with this child.
+    ///
+    /// Equivalent to the [`GetProcessId`][get-process-id] function.
+    ///
+    /// # Examples
+    ///
+    /// ```ignore
+    /// use create_process_w::Command;
+    ///
+    /// let mut command = Command::new("notepad.exe");
+    ///
+    /// if let Ok(child) = command.spawn() {
+    ///     match child.id() {
+    ///         Ok(id) => println!("Child's ID is {}", child.id());
+    ///         Err(err) => println!("Cannot get child's ID");
+    ///     } else {
+    ///         println!("notepad didn't start");
+    ///     }
+    /// }
+    /// ```
+    ///
+    /// [get-process-id]: https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-getprocessid
+    pub fn id(&self) -> Result<u32> {
+        // TODO: test this function on Windows
         unsafe {
-            let res = TerminateProcess(self.process_information.hProcess, 0);
+            let process_id = GetProcessId(self.process_information.hProcess);
```

Comment:
I thought hProcess was already the process id. I need to check the doc..

Reply:
Not really, `hProcess` is a handle to the process, `hThread` is an handle to the thread of the process.
The `PROCESS_INFORMATION` struct give two other fields: `dwProcessId` and `dwThreadId`.
https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/ns-processthreadsapi-process_information

Reply:
Why not use dwProcessId? It's also not the process id?

Reply:
It was really useless since i read this xD
But was an opportunity to go deeper with `windows-rs`, will by fix asap